compiler/dalgona/driver/Driver.cpp

```diff
@@ -59,7 +66,25 @@ int entry(const int argc, char **argv)
   if (arser["--analysis_args"])
     analysis_args = arser.get<std::string>("--analysis_args");
 
-  // TODO Run analysis
+  // Initialize python interpreter
+  py::scoped_interpreter guard{};
+
+  Dalgona dalgona;
+
+  // Initialize interpreter and operator hooks
+  dalgona.initialize(input_model_path);
+
+  // Run analysis
+  if (arser["--input_data"])
+  {
+    const auto input_data_path = arser.get<std::string>("--input_data");
```

Comment:
Q) is it OK not to check input data is valid?

Reply:
It is checked inside :) HDF5Importer will throw an exception if the input is invalid.